### PR TITLE
Fix: example set up

### LINKED ISSUE #1865: [Node.js] - develop branch - module not found @iota/wallet
## Bug description

When updating the instructions on how to get started with node.js for the `develop` branch, I keep getting `Error: Cannot find module '@iota/wallet'` when trying to run any example.

## Rust version

- Rust version: rustc 1.67.1 (d5a82bbd2 2023-02-07)

## Version

`develop`

## Hardware specification

- Operating system: Ubuntu 22.04
- RAM: 16
- Cores: 12
- Device: HP HP Pavilion Gaming Laptop 16-a0xxx

## Steps To reproduce the bug

1. `git clone git@github.com:iotaledger/wallet.rs.git`
2. `cd wallet.rs/bindings/nodejs/`
3. `npm i`
4. `npm run install`
5. `node examples/1-create-account.js`

or simply : ` git clone git@github.com:iotaledger/wallet.rs.git && cd wallet.rs/bindings/nodejs/
 && npm i && npm run install && node examples/1-create-account.js `

## Expected behaviour

The example should run,  and create an account.

## Actual behaviour

The example throws a `MODULE_NOT_FOUND` error  for @iota/wallet

## Errors

```plaintext
node:internal/modules/cjs/loader:958
  throw err;
  ^

Error: Cannot find module '@iota/wallet'
Require stack:
- /home/lucas/Documents/IOTA/tests/buildWalletNode/wallet.rs/bindings/nodejs/examples/1-create-account.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:955:15)
    at Module._load (node:internal/modules/cjs/loader:803:27)
    at Module.require (node:internal/modules/cjs/loader:1021:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/home/lucas/Documents/IOTA/tests/buildWalletNode/wallet.rs/bindings/nodejs/examples/1-create-account.js:6:38)
    at Module._compile (node:internal/modules/cjs/loader:1119:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1173:10)
    at Module.load (node:internal/modules/cjs/loader:997:32)
    at Module._load (node:internal/modules/cjs/loader:838:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/lucas/Documents/IOTA/tests/buildWalletNode/wallet.rs/bindings/nodejs/examples/1-create-account.js'
  ]
}

Node.js v18.8.0
```

### PR DESCRIPTION
**Pull request by Tuditi**
# Description of change

This PR separates code relevant to the examples and the bindings. To run an example you have to rename `.env.example` to `.env` and run the following code:

`cd wallet/bindings/nodejs && npm install && cd examples && npm install && node 1-create-account.js`

Alternatively, it is also possible to run `cd wallet/bindings/nodejs/examples && npm install && node 1-create-account.js`, but this rebuilds the bindings and takes a longer time (I cannot figure out why) without showing what happens in the terminal.

Once everything has been built it's possible to call the example from both the bindings folder and the examples folder.

## Links to any relevant issues

closes: #1865 

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

I ran the first example

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: documentation/docs/libraries/nodejs/how_to/_run_how_tos.mdx
Comment (lucas-tortora) [inline documentation/docs/libraries/nodejs/how_to/_run_how_tos.mdx]: Just in case, we've also updated these instructions on https://github.com/iotaledger/wallet.rs/pull/1917 to include the `npm i` from the examples directory.

File: documentation/docs/libraries/nodejs/how_to/_run_how_tos.mdx
Comment (lucas-tortora) [inline documentation/docs/libraries/nodejs/how_to/_run_how_tos.mdx]:
```suggestion
existing [`.env.example`](https://github.com/iotaledger/wallet.rs/blob/develop/wallet/bindings/nodejs/examples/.env.example) file. You can
```

File: documentation/docs/libraries/nodejs/how_to/_run_how_tos.mdx
Comment (lucas-tortora) [inline documentation/docs/libraries/nodejs/how_to/_run_how_tos.mdx]:
```suggestion
do this running the following command from the `wallet/bindings/nodejs/examples` directory:
```

File: documentation/docs/libraries/nodejs/how_to/_run_how_tos.mdx
Comment (lucas-tortora) [inline documentation/docs/libraries/nodejs/how_to/_run_how_tos.mdx]:
```suggestion
After you have followed the instructions to [build the library](#build-the-library), you have to run the following command in `wallet/bindings/nodejs/examples`:
```

**Review by lucas-tortora — CHANGES_REQUESTED** [review]
Thanks for getting to this so quickly :) !!! 

Some minor URL/directory issues in the docs. 

Btw, there is another PR to add the same changes to the docs. https://github.com/iotaledger/wallet.rs/pull/1917

File: documentation/docs/libraries/nodejs/how_to/_run_how_tos.mdx
Comment (Tuditi) [reply documentation/docs/libraries/nodejs/how_to/_run_how_tos.mdx]: Maybe we can wait until #1917 is merged and then I'll rebase this branch :relaxed:

File: documentation/docs/libraries/nodejs/how_to/_run_how_tos.mdx
Comment (Thoralf-M) [inline documentation/docs/libraries/nodejs/how_to/_run_how_tos.mdx]: This is basically duplicated with what's in `### Install Dependencies` below

**Review by Thoralf-M — APPROVED** [review]

**Review by lucas-tortora — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**
